src/coord/src/coord.rs

```diff
@@ -2582,7 +2582,10 @@ where
 
 fn is_cat_index(id: &GlobalId) -> bool {
     // TODO(justin): do something better here?
-    CatalogView::all_views().iter().any(|v| v.index_id() == *id)
+    BUILTINS.values().any(|b| match b {
```

Comment:
super minor nit but just want to confirm my understanding -- you could use the tables iterator here right?

Reply:
Oops, yes! I added that iterator after I rewrote this method. Thanks, updated.